bangazonapi/models/like.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+from .customer import Customer
+from .product import Product
+
+class Like(models.Model):
+
+    customer = models.ForeignKey(Customer, on_delete=models.DO_NOTHING,)
+    product = models.ForeignKey(Product, on_delete=models.DO_NOTHING,)
```

Comment:
Not a blocker, but are you wanting likes to remain in the db even if their customer or product is deleted?